bedrock/firefox/tests/test_base.py

```diff
@@ -1065,3 +1065,39 @@ def test_whatsnew_search_tour_oldversion(self):
         # if there's no oldversion parameter, show no tour
         response = self.client.get(self.url, HTTP_USER_AGENT=self.user_agent)
         self.assertNotIn(self.expected, response.content)
+
+
+class TestHelloStartView(TestCase):
+    def setUp(self):
+        self.user_agent = ('Mozilla/5.0 (Macintosh; Intel Mac OS X 10.8; rv:35.0) '
+                      'Gecko/20100101 Firefox/35.0')
+        self.url = reverse('firefox.hello.start', args=['35.0'])
+
+    @override_settings(DEV=True)
+    @patch.dict(product_details.firefox_versions,
+                LATEST_FIREFOX_VERSION='16.0')
```

Comment:
I'm not too sure? I'm just copying how other tests in this file have been done :smile: